### DISCUSSION TITLE
Provide a 'stop' command in CLI CKAN

### DISCUSSION THREAD

**Discussion by c24b** [ckan/ckan#6463] [Ideas]
As ckan cli  provides a run command:

`ckan -c /etc/ckan/default/ckan.ini run`

Possibility of stopping ckan from running in background should be offer such as:

`ckan -c /etc/ckan/default/ckan.ini  stop`

This would clear the supervisor process and free the occupied ports.
```
#stop the uwsgi server
sudo supervisorctl stop ckan-worker:ckan-worker-00 ckan-datapusher:ckan-datapusher-00 ckan-uwsgi:ckan-uwsgi-00
# remove sites from nginx
sudo rm /etc/nginx/sites-enabled/ckan 
```

as it seems that cmd run makes all this by default
this cmd command `stop` should be the clean way to reverse it

**Comment by frafra** [comment]
To stop CKAN should be enough to send a SIGTERM/SIGKILL to it, so I fail to see why there should be a custom command for that in the CKAN CLI. It should be handled by the process manager (supervisord, systemd, sysvinit, ...) or by the containers' orchestrator.
